lib/blockchain_double.js

```diff
@@ -1042,6 +1042,12 @@ BlockchainDouble.prototype.increaseTime = function(seconds) {
   return this.timeAdjustment;
 };
 
+BlockchainDouble.prototype.decreaseTime = function(seconds) {
```

Comment:
Any reason not to let `increaseTime` accept negative values? That'd be one fewer public method.

Reply:
mainly because "increaseTime" as read by a developer would indicate time is going up, and a negative value on the variable is easy to mistake on a glance, while reading right to left of a function is pretty solid.

increaseTime(forwardValue)
increaseTime(-forwardValue)

vs

increaseTime(forwardValue)
decreaseTime(backwardsValue)

Reply:
plus reading pure function definitions in the readme (which I didn't update in this PR) would be immediately visible, while you'd have to create creating an extra note surrounding a negative value being acceptable in the alternative.